libraries/core-react/src/components/Button/Button.stories.tsx

```diff
@@ -17,6 +19,12 @@ const FullWidthWrapper = styled.div`
   grid-gap: 16px;
 `
 
+const GroupWrapper = styled.div`
+  margin: 32px;
+  display: grid;
+  grid-template-columns: repeat(4, fit-content(100%));
```

Comment:
Don't think we need this css? Maybe you can extend the `Wrapper` with styled and set `grid-gap: 0` if you need that margin for the story,

Reply:
Great, thanks